### PR TITLE
Scaffold single

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: partials/singles/single-post-fm.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<div class="row">
+  <div class="col col24">
+    <h4>FM</h4>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col col12">
+    <h1 class="js-fix-widows"><?php the_title(); ?></h1>
+    // social sharing
```
Comment (mixcas) [inline partials/singles/single-post-fm.php:14]: what kind of comment is this?

Comment (varffs) [inline partials/singles/single-post-fm.php:14]: placeholder as in not actually a code comment but to be seen

File: partials/singles/single-post-fm.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+<div class="row">
+  <div class="col col24">
+    <?php
+      if (!empty($meta['_cmb_sc'][0])) {
+    ?>
+        <iframe src="https://w.soundcloud.com/player/?url=<?php echo urlencode($meta['_cmb_sc'][0]); ?>" width="100%" height="200" scrolling="no" frameborder="no"></iframe>
+
+      <?php
+        if ($meta['_cmb_is_resonance'][0]) {
```
Comment (mixcas) [inline partials/singles/single-post-fm.php]: this is erroring out
`Undefined index: _cmb_is_resonance in /var/www/novaramedia-com/wp-content/themes/novaramedia-com/partials/singles/single-post-fm.php on line 36`
when the field is unchecked

File: partials/singles/single-post-tv.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  </div>
+</div>
+
+<div class="row">
+  <div class="col col20">
+    <?php
+      if (!empty($meta['_cmb_utube'])) {
+    ?>
+      <div class="u-video-embed-container">
+        <iframe class="youtube-player" type="text/html" src="http://www.youtube.com/embed/<?php echo $meta['_cmb_utube'][0]; ?>?autohide=2&amp;modestbranding=1&amp;origin=http://novaramedia.com&amp;showinfo=0&amp;theme=light&amp;rel=0"></iframe>
```
Comment (mixcas) [inline partials/singles/single-post-tv.php:29]: it thought we were going to use YouTube JavaScript Player API for this

File: partials/singles/single-post-fm.php (hunk 2 above)
Comment (varffs) [inline partials/singles/single-post-fm.php]: hmm yes i suppose on posts that havent been updated and therefore saved the false value will be empty. yes have to check for that for historical posts

Comment (mixcas) [inline partials/singles/single-post-fm.php]: maybe in single is fine like that
is there another youtube related functionality in the single beyond embedding the vid?

File: partials/singles/single-post-tv.php (hunk 3 above)
Comment (mixcas) [inline partials/singles/single-post-tv.php:29]: maybe in single is fine like that
is there another youtube related functionality in the single beyond embedding the vid?

File: single.php [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  <!-- main posts loop -->
+  <section id="post" class="container margin-bottom-basic">
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+
+    $categories = get_the_category();
+
+    function onlyTopLevel($var) {
```
Comment (mixcas) [inline single.php]: shouldn't this be  in the functions files?

Comment (varffs) [inline single.php]: i dunno. its only for that context but sure

File: partials/singles/single-post-tv.php (hunk 3 above)
Comment (varffs) [inline partials/singles/single-post-tv.php:29]: what do we need to do to use the API? surely just the add an ID to the embed?

File: single.php (hunk 4 above)
Comment (mixcas) [inline single.php]: yeah i think all functions should be in functions

File: partials/singles/single-post-tv.php (hunk 3 above)
Comment (varffs) [inline partials/singles/single-post-tv.php:29]: its only the playlist section that is going to load in other videos, and load next after the end of a video, etc. that is shown on the home page and some archives only I think

Comment (mixcas) [inline partials/singles/single-post-tv.php:29]: yeah a field for the vid id or full url

Comment (varffs) [inline partials/singles/single-post-tv.php:29]: ? that field does have the vid id?

**Comment by varffs** [comment]
fixed the issues. Leaving player alone until a branch/feature that focuses on that UX

**Pull request merged**